### DISCUSSION TITLE
Playtime: FX plugins on sends (on return tracks) they lag quite much even with 1 track and 1 send/return

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1252] [General]
When I load FX’s as sends (on return tracks) they lag quite much in the Playtime 2, not sure why when my PC is 128GB ram and 24 core i9 intel processor… and it is happening when loading only 1 instrument track and 1 send/return track. Crazy isn’t it? It seems not to cause any lag in the FX chain for the same track (not using sends/returns, only putting on the same track all the plugins), but for sends/returns it makes it lag quite much

**Comment by helgoboss** [comment]
Hi. At first, please try the same in 2.16.10 and see if the issue persists. If yes, I need a minimal project file using only built-in FX along with some instructions how to reproduce this issue. Thanks.

**Reply by joungle123** [reply]
The issue is solved for 1 track which has 1 send/return with the version of 2.16.10. However I tried scenario of 1 track which has 6 sends/returns and they lag quite much .. Let me know if you need any additional info from me

**Reply by helgoboss** [reply]
I need a minimal project then using only built-in FX along with some instructions how to reproduce this issue. Thanks.

**Reply by joungle123** [reply]
Here I created a project with 1 track and 7 sends. First track is instrument track (reaper synth) and other tracks are sends of track 1, those are reverbs (reaper reverbs). Just try to move slider named 'Wet' in the reverb (e.g. Send/Return track 8) and you will see how it lags. It is waaaaay more visible and apparent in 3rd party plugins having designed UI. I noticed this by using Valhalla reverb which is actually very CPU light ...
[1 track - 7 sends.zip](https://github.com/user-attachments/files/17194654/1.track.-.7.sends.zip)

**Reply by helgoboss** [reply]
Hi, I tried this but there's no lag here. I'm on macOS though, maybe that makes a difference. Even tried replacing the ReaVerb in track "Column 8" with Valhalla Room ... everthing smooth when I move the MIX slider.

1. Could you make a short screen recording showing what kind of lag that is? I suppose it's not an audio lag but a GUI lag!?
2. I don't have Klinke and DrivenByMoss installed. Maybe makes a difference, too? Have you tried in a clean portable REAPER installation?
3. Does the issue occur when you remove the Playtime track (and play the clip as a normal REAPER item instead)?

**Reply by joungle123** [reply]
@helgoboss It seems that with the new (latest) version of the release the issue does not appear anymore, thank you!
